src/Agent.Worker/Handlers/ProcessHandler/ProcessHandlerHelper.cs

```diff
@@ -132,7 +133,13 @@ private static int FindEnclosingIndex(string input, int targetIndex)
                     {
                         if (enableAudit && !enableValidation)
                         {
-                            context.Warning(StringUtil.Loc("ProcessHandlerInvalidScriptArgs"));
+                            var issue = new Issue
+                            {
+                                Type = IssueType.Warning,
+                                Message = StringUtil.Loc("ProcessHandlerInvalidScriptArgs"),
+                            };
+                            issue.Data.Add("auditAction", "1"); // ShellTasksValidation = 1
```

Comment:
I keep it hard-coded as vss-api-netcore with constants is not published yet.
I'll update it once new package version will be available